helm/policies-azure/tests/abs/ensure.py

```diff
@@ -0,0 +1,119 @@
+import yaml
+from functools import partial
+import time
+import random
+import string
+from textwrap import dedent
+
+import pytest
+from pytest_kube import forward_requests, wait_for_rollout, app_template
+
+import logging
+LOGGER = logging.getLogger(__name__)
+
+cluster_name = "test-cluster"
+release_version = "20.0.0"
+cluster_apps_operator_version = "2.0.0"
+capa_version = "capi"
```

Comment:
maybe we can call this `watch_label` or something generic?

Reply:
Sure

Reply:
Done now - going with this. I can fix anything else in a followup :)